projects/angular/src/forms/common/wrapped-control.ts

```diff
@@ -194,26 +194,29 @@ export class WrappedFormControl<W extends DynamicWrapper> implements OnInit, DoC
   }
 
   private getAriaDescribedById(helpers: Helpers): string | null {
-    let suffix = CONTROL_SUFFIX.HELPER;
-
-    if (helpers.showInvalid) {
-      suffix = CONTROL_SUFFIX.ERROR;
-    } else if (helpers.showValid) {
-      suffix = CONTROL_SUFFIX.SUCCESS;
-    }
-
+    let elementId;
```

Comment:
```suggestion
    let elementId: string;
```

missing type annotation